test/parallel/test-worker-nearheaplimit-deadlock.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const common = require('../common');
```

Comment:
Test is failing on Raspberry Pi devices. Does it make sense to skip on those devices? If so, maybe this?

```suggestion
if ((process.config.variables.arm_version === '6') ||
  (process.config.variables.arm_version === '7')) {
  common.skip('Skipping on low-memory armv6 and armv7 bots');
}
```

I'm not sure the reason I provide in `common.skip()` above is accurate. Update if not, of course.